arthur_bench/run/testsuite.py

```diff
@@ -46,7 +47,7 @@ class TestSuite:
     def __init__(
         self,
         name: str,
-        scoring_method: Union[str, ScoringMethod],
+        scoring_method: Union[str, Scorer],
```

Comment:
Maybe update docstring for this since it can take a method or scorer?

Reply:
good catch - updated!